charts/jenkins/values-casc-jobs.yaml.gotmpl

```diff
@@ -84,6 +84,7 @@ controller:
         - script: > {{ tpl (readFile "jobs/nuxeo/lts/nuxeo-jsf-ui.groovy.gotmpl") . | nindent 12 }}
         - script: > {{ tpl (readFile "jobs/nuxeo/lts/nuxeo.groovy.gotmpl") . | nindent 12 }}
         - script: > {{ tpl (readFile "jobs/nuxeo/lts/promote-nuxeo-server.groovy.gotmpl") . | nindent 12 }}
+        - script: > {{ tpl (readFile "jobs/nuxeo/lts/test-nuxeo-server-against-es7.17.groovy.gotmpl") . | nindent 12 }}
```

Comment:
Please try to keep alphabetical sort for readability.
If we agree about nuxeo-es-7.17, then it will go after nuxeo-benchmark.